binr/ragg2/ragg2.c

```diff
@@ -300,36 +300,36 @@ static struct cEnv_t* set_cEnv(const char *arch, const char *os, int bits) {
 			if (bits == 32) {
 				cEnv->CFLAGS = strdup ("-fPIC -fPIE -pie -fpic -m32");
 				if (cEnv->CFLAGS == NULL) {
-					eprintf("Not enough memory for CFLAGS");
+					eprintf ("Not enough memory for CFLAGS");
 					goto fail;
 				}
 				cEnv->LDFLAGS = strdup ("-fPIC -fPIE -pie -fpic -m32");
 				if (cEnv->LDFLAGS == NULL) {
-					eprintf("Not enough memory for LDFLAGS");
+					eprintf ("Not enough memory for LDFLAGS");
 					goto fail;
 				}
 			} else {
 				cEnv->CFLAGS = strdup ("-fPIC -fPIE -pie -fpic -m64");
 				if (cEnv->CFLAGS == NULL) {
-					eprintf("Not enough memory for CFLAGS");
+					eprintf ("Not enough memory for CFLAGS");
 					goto fail;
 				}
 				cEnv->LDFLAGS = strdup ("-fPIC -fPIE -pie -fpic -m64");
 				if (cEnv->LDFLAGS == NULL) {
-					eprintf("Not enough memory for LDFLAGS");
+					eprintf ("Not enough memory for LDFLAGS");
```

Comment:
No need to error that precisely. When allocations fail you usually want to fail fast, crash. If that alloc fails means so many other mallocs will fail so is better to quit asap. Printfs sill result in so much bigger binaries and usually in lot of error printfs

Reply:
Then should I use exit (1) immediately instead of goto fail ?